src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
```

Comment:
You should store all colors in variables in a separate file. This will give you the advantage that if someday your client comes and wants to change all the colors of the borders from red to black, you will be able to do it only in one place and not in the entire project

Plus check https://github.com/mate-academy/layout_catalog/blob/master/checklist.md

11. [SASS] - use variables for the main values so that you'll be able to reuse them and give them descriptive names. But don't overuse them, don't create variable for the value that's used just once.